tests/test_facebook_bookmarks.py

```diff
@@ -196,11 +196,6 @@ def bookmarks_test(self, expected_streams):
 
```

Comment:
@harshpatel4crest Can you add a check in the integration test case to check the format of the `date_start` and `date_stop`. You can verify the format for only one record.. That would be enough.

Reply:
Added the check for the date.